aispace2/jupyter/stepdomwidget.py

```diff
@@ -73,6 +76,12 @@ def _validate_line_width(self, proposal):
         line_width = proposal['value']
         return min(1, line_width)
 
+    @validate("text_size")
+    def _validate_line_width(self, proposal):
```

Comment:
_validate_text_size might be a better name for this function

Reply:
done